clusterman/migration/worker.py

```diff
@@ -112,14 +117,23 @@ def _drain_node_selection(
     """
     nodes = manager.get_node_metadatas(AWS_RUNNING_STATES)
     selected = sorted(filter(selector, nodes), key=worker_setup.precedence.sort_key)
+    if not selected:
+        return True
+    monitoring_info = {"cluster": manager.cluster, "pool": manager.pool}
+    node_drain_counter = get_monitoring_client().create_counter(SFX_NODE_DRAIN_COUNT, monitoring_info)
+    job_timer = get_monitoring_client().create_timer(SFX_MIGRATION_JOB_DURATION, monitoring_info)
+    node_uptime_gauge = get_monitoring_client().create_gauge(SFX_DRAINED_NODE_UPTIME, monitoring_info)
     chunk = worker_setup.rate.of(len(nodes))
     logger.info(f"{len(selected)} nodes of {manager.cluster}:{manager.pool} will be recycled")
+    job_timer.start()
     for i in range(0, len(selected), chunk):
         start_time = time.time()
         selection_chunk = selected[i : i + chunk]
         for node in selection_chunk:
             logger.info(f"Recycling node {node.instance.instance_id}")
             manager.submit_for_draining(node)
+            node_uptime_gauge.set(node.instance.uptime.total_seconds(), {"ip": node.instance.ip_address})
```

Comment:
I think we should add more general label to create dashboard/statistics instead of ip. for ex: cluster/pool, OS version, and etc.

Reply:
Node and pool will be included already by the base setup of the gauge. I thought having the node IP could be useful to backtrack anomalies, but if you don't think that's the cause I can just remove that extra info.

Reply:
Got it, Yeah I think we should remove id. Because it may cause high cardinality in TSDB